README.md

```diff
@@ -22,6 +22,19 @@ config :movie_wager_api, MovieWagerApi.Repo,
 Similarly, you will want to follow the same steps for `test.secret.example.exs`.
 The secret files are not checked into source control, so don't worry about your credentials being shared.
 
+## Setting up Twitter Authentication
+
+The `dev.secret.exs` and `test.secret.exs` files also include information for the twitter OAuth API.
```

Comment:
:thinking: possibly. I can add a "ask Scott for these credentials" or link to the twitter dev docs if you want to roll your own.